actions/team_actions.test.js

```diff
@@ -96,8 +96,8 @@ describe('Actions.Team', () => {
     });
 
     test('addUserToTeamFromInvite', async () => {
```

Comment:
The name of this test should change:

```suggestion
    test('addUserToTeam', async () => {
```